api/src/main/java/org/apache/gravitino/authorization/Privilege.java

```diff
@@ -39,6 +40,14 @@ public interface Privilege {
    */
   Condition condition();
 
+  /**
+   * If a metadata object supports a privilege, the privilege will return true. Otherwise, false.
```

Comment:
a privilege -> this privilege or the privilege. Then the method will return true otherwise false.

Reply:
I believe 

If the privileges support metadata with the specified type, the method will return true otherwise false.

Reply:
OK.